### PR TITLE
Add rpcbind service check

### PR DESCRIPTION
**Pull request by AlynxZhou**
Add rpcbind service check (enable, config and test with nfs-server).

- Related ticket: https://progress.opensuse.org/issues/55007
- Needles: n/a
- Verification run: 
    + Migration service check(running): http://openqa.suse.de/tests/3293447
    + Console rpcbind check: http://openqa.suse.de/tests/3293288

### CODE REVIEW

File: lib/services/rpcbind.pm [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Package for dhcp service tests
```
Comment (coolgw) [inline lib/services/rpcbind.pm]: package for rpc, not for dhcp

Comment (AlynxZhou) [reply lib/services/rpcbind.pm]: Fixed.

File: lib/services/rpcbind.pm [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        install_service();
+        config_service();
+        enable_service();
+        start_service();
+    }
+    check_service();
+    check_function();
+}
+
+1;
```
Comment (coolgw) [inline lib/services/rpcbind.pm]: should we also update the ./tests/console/rpcbind.pm using the func of lib/services/rpcbind.pm?

Comment (AlynxZhou) [reply lib/services/rpcbind.pm]: I have updated a commit and now I am re-running related test.

File: tests/console/rpcbind.pm [hunk 3]
```diff
@@ -15,37 +15,22 @@ use base 'consoletest';
...
 use testapi;
 use utils qw(systemctl zypper_call);
+use services::rpcbind;
 
 sub run {
     my ($self) = @_;
     $self->select_serial_terminal();
-    zypper_call 'in rpcbind nfs-kernel-server';
+    services::rpcbind::install_service();
     assert_script_run 'rpm -q rpcbind';
```
Comment (coolgw) [inline tests/console/rpcbind.pm]: create func for this line.

**Comment by coolgw** [comment]
@plusky could you help review this?

**Pull request merged**